### DISCUSSION TITLE
Pagination: remove 'all' option && overwrite pagination blade file

### DISCUSSION THREAD

**Discussion by m-pernkopf** [filamentphp/filament#9785] [General]
Dear community,

I'd like to remove the 'all' option from the ressource pagination dropdown; I have a ressource with nearly 10k entries which breaks my backend, when choosing the 'all' option.
I couldn't figure out how to disable this option in any config file or similar, I also already tried to overwrite the 

_records-per-page-selector.blade.php (from vendor/filament/tables/resources/views/components/pagination)_ 

in my own 

_records-per-page-selector blade file in resources/views/vendor/filament-tables/components/pagination_ 

but it seems that this blade file is not loaded correctly (I did some modifications but backend is loading the original blade file from vendor/filament folder).

Any hint will be appreciated

Regards, Michael

**Comment by leandrocfe** [comment]
Are you using Filament V3?
https://filamentphp.com/docs/3.x/tables/advanced#customizing-the-pagination-options

**Reply by m-pernkopf** [reply]
We are using Filament V2.
Didn't find a paginated Function (as pointed out in your link), but in V2 there is a function

protected function getTableRecordsPerPageSelectOptions(): array 
    {
        return [10, 25, 50, 100];
    }

which does exactly what I need in my Resource List Page.

**Reply by leandrocfe** [reply]
Ahh ok, you can also publish and edit the config
`php artisan vendor:publish --tag=tables-config`
https://github.com/filamentphp/filament/blob/2.x/packages/tables/config/tables.php
